bootstrap_panicdisabled.go

```diff
@@ -0,0 +1,20 @@
+//go:build !onscreenpanic
+// +build !onscreenpanic
+
+package europi
+
+import (
+	"github.com/awonak/EuroPiGo/hardware"
+	"github.com/awonak/EuroPiGo/hardware/hal"
+)
+
+func init() {
+	hardware.OnRevisionDetected <- func(revision hal.Revision) {
+		switch revision {
+		case hal.RevisionUnknown:
```

Comment:
[required] Since we have an identifier for the EuroPi Prototype and we know it doesn't have a screen, lets add it here.
```suggestion
		case hal.RevisionUnknown, hal.EuroPiProto:
```